content/gotraining.slide

```diff
@@ -425,6 +429,53 @@ As part of all labs
 - Create a PR following the [[https://github.com/anz-bank/go-training#pre-pr-checklist][PR checklist]]
 
 
+* PR from own fork - initial setup
+
+Issue PRs from `feature` branch of your own fork to `master` on upstream:
+
+- Login to github.com
+- Click `Fork` in the top right corner of [[https://github.com/anz-bank/go-training-slides]]
+- Set up the following remotes (`USERNAME` is your GitHub username):
+
+	language:none
+	git remote -v
```

Comment:
Perhaps add the git command for adding the upstream remote and having master track upstream/master:

`git remote add upstream https://github.com/anz-bank/go-training`
`git remote update`
`git branch -u upstream/master master`
`git pull`